### ISSUE TITLE
Storage of non-scientific data on Instrument PCs

### ISSUE THREAD

**Issue by KathrynBaker** [ISISComputingGroup/IBEX#8321]
As a supporter of the IBEX systems, I would like to confirm and ensure that our non-scientific data retention policy is updated and enacted to limit the amount of work being done in an ad hoc manner.

### Acceptance Criteria
- [x] The policies in https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Data-Generation-and-Storage are checked and confirmed - this would be a meeting for an hour
- [x] Tickets are created to automate that which can be automated
- [x] Process documents are updated to highlight processes that cannot be automated, and these are communicated to the group

### Time in Planning
01:13:14 02/05/24

**Comment by ChrisM-S** [comment]
May be useful to be aware of this too https://www.isis.stfc.ac.uk/Pages/Data-Policy.aspx - this document morphs a bit over time, it used to have 10 years as a limit for raw data to be kept but by careful reading it looks like the 10 year period is still there as a minimum and does apply to _metadata_ so it may be that we should be increasing some of these to 10 years (or at least 10 years)?

**Comment by GRyall** [comment]
Following our discussion, I have updated (and moved - sorry, didn't realise the impact that would have) the wiki page: https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Data-Generation-and-Storage-on-Instrument-PCs-(NDX's).

I have also created the following tickets:
- #8360 
- #8361
- #8362 
- #8363
- #8364 
- #8365 

I believe that completes the ticket. If I have missed something from our discussion, I will update the wiki/tickets as needed.
